### PR TITLE
Silence some qt warnings which aren't our fault

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

File: src/app/main.cpp [hunk 1]
```diff
@@ -405,7 +405,9 @@ void myMessageOutput( QtMsgType type, const QMessageLogContext &, const QString
            msg.contains( QLatin1String( "QGestureManager::deliverEvent" ), Qt::CaseInsensitive ) ||
            msg.startsWith( QLatin1String( "libpng warning: iCCP: known incorrect sRGB profile" ), Qt::CaseInsensitive ) ||
            msg.contains( QLatin1String( "Could not add child element to parent element because the types are incorrect" ), Qt::CaseInsensitive ) ||
-           msg.contains( QLatin1String( "OpenType support missing for" ), Qt::CaseInsensitive ) )
+           msg.contains( QLatin1String( "OpenType support missing for" ), Qt::CaseInsensitive ) ||
+           msg.contains( QLatin1String( "Wayland does not support" ), Qt::CaseInsensitive ) ||
```
Comment (m-kuhn) [inline src/app/main.cpp:409]: Qt5 only?

Comment (nyalldawson) [reply src/app/main.cpp:409]: Not sure, but there's a range of messages that are shown like "Wayland does not support QWidget::raise", etc. basically just noise, we can't fix those. 

I'd run x11 if I could, but fedora 40 KDE removed that . There's no winners here 🤷‍♂️

**Review by m-kuhn — APPROVED** [review]
It won't hurt to silence these in any case. We might want to add a comment to reevaluate all of these with Qt6?
I.e. completely remove all these filters at some point.

**Comment by nyalldawson** [comment]
> We might want to add a comment to reevaluate all of these with Qt6?
I.e. completely remove all these filters at some point.

Done!

**Pull request merged**